From: Joel Selvaraj <foss@joelselvaraj.com>
Date: Fri, 13 Jun 2025 22:42:16 -0500
Subject: arch: arm64: dts: qcom: sdm845-oneplus: add q6voicedai reg

This makes it similar to the other dais. Noticed in sdm660 repo.  But I am not sure if it's a functional requirement. Anyways let's specify it.

Signed-off-by: Joel Selvaraj <foss@joelselvaraj.com>
---
 arch/arm64/boot/dts/qcom/sdm845-oneplus-common.dtsi | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm64/boot/dts/qcom/sdm845-oneplus-common.dtsi b/arch/arm64/boot/dts/qcom/sdm845-oneplus-common.dtsi
index 3697ce6..d07f51a 100644
--- a/arch/arm64/boot/dts/qcom/sdm845-oneplus-common.dtsi
+++ b/arch/arm64/boot/dts/qcom/sdm845-oneplus-common.dtsi
@@ -942,6 +942,12 @@ dai@5 {
 	};
 };
 
+&q6voicedai {
+	dai@1 {
+		reg = <VOICEMMODE1>;
+	};
+};
+
 &qupv3_id_1 {
 	status = "okay";
 };
